src/borough/borough.repository.ts

```diff
@@ -183,12 +191,31 @@ export class BoroughRepository {
             ST_Intersects(${communityDistrict.mercatorFill}, ${capitalProject.mercatorFillMPoly})
             OR ST_Intersects(${communityDistrict.mercatorFill}, ${capitalProject.mercatorFillMPnt})`,
         )
+        .leftJoin(
+          capitalCommitment,
+          and(
+            eq(capitalCommitment.capitalProjectId, capitalProject.id),
+            eq(capitalCommitment.managingCode, capitalProject.managingCode),
+          ),
+        )
+        .leftJoin(
+          capitalCommitmentFund,
+          eq(capitalCommitmentFund.capitalCommitmentId, capitalCommitment.id),
+        )
         .where(
           and(
             eq(communityDistrict.id, communityDistrictId),
             eq(communityDistrict.boroughId, boroughId),
+            eq(capitalCommitmentFund.category, "total"),
           ),
         )
+        .groupBy(
```

Comment:
What's the purpose of this `groupBy`? I wouldn't necessarily expect to see this new clause for adding a property to the tiles.

Reply:
Without the groupBy, we would get an entry for every commitment that belongs to the capital project. Also, the value of the "commitmentsTotal" property would be the total for only that commitment.

With the groupBy, we get a single entry for each capital project. Also, the value of the "commitmentsTotal" property is the summation of all commitments for that capital project.